sw/device/tock/boards/opentitan/tock_dependencies/Cargo.lock

```diff
@@ -0,0 +1 @@
+/home/svt/development/repos/opentitan/sw/device/tock/boards/opentitan/tock_dependencies/Cargo_remote.lock
```

Comment:
You need to update the `.gitignore` to ignore this file.

Reply:
> I don't think we should be nesting cargo packages like you have with boards/opentitan/earlgrey and boards/opentitan/earlgrey/peripherals (and also boards/opentitan/tock_dependencies).

We don't have to nest them, I am happy to pull it out outside `board/` directory, but could you please why we shouldn't do it in your opinion in more detail.

> How are sw/device/tock/boards/opentitan/src/*.rs built? Don't they need a Cargo.toml?

They do. It is the `Cargo.toml` in the `boards/opentitan` directory.

Reply:
> They do. It is the `Cargo.toml` in the `boards/opentitan` directory. However, I couldn't find it in the committed files, I think it's maybe to do with the fact that the symlink with this name was already in git. I will investigate this.

It is because `Cargo.toml`used to be a symlink and was in `.gitignore`. Nice catch, thank you!